src/main/java/reposense/RepoSense.java

```diff
@@ -108,6 +108,7 @@ public static void main(String[] args) {
             // help message was printed by the ArgumentParser; it is safe to exit.
         }
 
+        LogsManager.moveLogFileToOutputFolder();
         LogManager.getLogManager().reset();
```

Comment:
I think with the new implementation, this line can be removed since you already close the relevant file handler before deleting the logs folder. This was previously added due to the .lck file lingering and I wasn't sure what was the cause but it seems you've addressed it.